lib/views/Deposit/DepositCard.tsx

```diff
@@ -310,22 +310,55 @@ const HelpLink = () => {
   )
 }
 
-interface ExternalLinkProps {
-  prizePool: PrizePool
+const purple_default = '4c249f'
```

Comment:
Could you pull this out into a separate location. A new file in `src/utils/services` makes sense to me.